app/javascript/components/pages/__test__/Header.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Header from "../../components/Header";
+
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('When Header loads', () => {
+    it('displays navlink when signed in',() => {
+        const header = shallow(<Header logged_in={true}
+            new_user_route="new_user_route"
+            sign_in_route="sign_in_route"
+            sign_out_route="sign_out_route"/>);
+        const nav = header.find('NavLink');
```

Comment:
Is there specific text for this NavLink? It'd be good to assert on it, in case someone changes on accident, a test should break